cli/args/flags.rs

```diff
@@ -10852,4 +10983,27 @@ mod tests {
 Usage: deno repl [OPTIONS] [-- [ARGS]...]\n"
     )
   }
+
+  #[test]
+  fn test_allow_import_host_from_url() {
```

Comment:
Can you add another unit test that passes `--allow-import` with value to show that it will override the defaults?

Reply:
There is a `specs::permissions::allow_import::builtin_host_replaced` integration test.